src/components/helpers/index.js

```diff
@@ -0,0 +1,12 @@
+const _ = {
```

Comment:
you need to transpile this file too when building, it's currently skipped as it does not end in .jsx... I think the easiest way is to rename this file src/components/helpers.jsx and then all will be ok. Up to now everything in the components folder has been a jsx file
install this branch into tripapp to see what I mean.